### ISSUE TITLE
No file chosen Import data (EARL as JSON-LD)

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#334]
WCAG-EM has a BETA mode to "Import data (EARL as JSON-LD)" https://www.w3.org/WAI/eval/report-tool/evaluation/audit-sample

> Add data from other (automated) tests [More info on importing EARL as JSON](https://github.com/w3c/wai-wcag-em-report-tool/wiki/How-to-use-the-%E2%80%9CAdd-data-from-other-(automated)-tests%E2%80%9D-functionality)

This would facilitate the process of filling in the reports. 

Would be great to integrate well with Microsoft's Accessibility Insights too:
https://github.com/microsoft/accessibility-insights-web/issues/4331

**Comment by 970Design** [comment]
I'm curious what tooling you use to generate your EARL data with.  It looks really promising to streamline this process, but haven't found many ways to actually create that data.

**Comment by mgifford** [comment]
Yes..  This is a problem. It is possible that EARL is overly-complicated for the task at hand. Could also be that it was just that everyone is more invested in finding something that works best for them with their use case. 

At this point you can't pull in data from other machine readable formats. I do not know if there are yet any good approaches to create and manage this data. 

There should be..
